main.py

```diff
@@ -21,7 +21,10 @@ def kktix_count(web):
     r = requests.get(web)
     soup = BeautifulSoup(r.text, "html.parser")
     ticket_count = soup.find("span", class_="info-count")
-    return ticket_count.text
+    if ticket_count != None:
```

Comment:
Is this statement more clear and straightforward?

```
if ticket_count:
    return "N/A"
else:
    return ticket_count.text
```

`!= None` is not comprehensive. For example, empty string may be also what you want to avoid.